.gitignore

```diff
@@ -141,3 +141,7 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+#migrations/
+
+app/mac_fucks_around_with_routes.py
```

Comment:
😛

It would probably be better to do this type of work in a separate branch and not push it to master instead of adding it to the .gitignore